### ISSUE TITLE
Back gesture on Android sometimes stops working as of latest update (1.74.0)

### ISSUE THREAD

**Issue by TinaRussell** [bluesky-social/social-app#3344]
**Describe the bug**
Ever since I last updated the Android app (to version 1.74.0), the "back" gesture on my phone (swiping left from the right edge of the screen) no longer works in Bluesky. It still works in other apps. (I am using a Moto G Power 2020, Android version 11.)

**To Reproduce**

1. Swipe left from the right edge of the screen, anywhere in the app.

**Expected behavior**

The app should return to the previous screen (such as going back to the timeline when viewing an individual post), or return to the home screen (when on the main timeline). This is what happened in previous versions, and what still happens on other apps.

(What actually happens, now, is that the back gesture animation appears—a thin, left-pointing arrowhead slides in from the right—but nothing else happens.)

**Screenshots**

![Screenshot_20240322-165923](https://github.com/bluesky-social/social-app/assets/31976617/9a6a61ef-7f3d-4efd-af2b-08547175e7d3)
In this image, on my main timeline, you can see an arrowhead pointing left on the upper-right side of the screen (that's where I was doing the back gesture), but nothing is happening.

**Details**

 - Platform:  Android
 - Platform version: 11
 - App version: 1.74.0

**Additional context**

I'm not _entirely_ sure this is a problem with version 1.74.0 specifically—I'm not sure what the current version was when I last updated—but it's certainly my best guess, as I try to update the app fairly often.

**Comment by haileyok** [comment]
Does this happen on a fresh launch or is it after a bit of use? We found an issue with the back button navigation after discarding a draft, which is probably what is happening here.

- Open composer
- Type a bit of a message
- Cancel and hit Discard
- Back button stops working

Mind seeing if that same thing happens here? Appreciate it!

**Comment by TinaRussell** [comment]
> Does this happen on a fresh launch or is it after a bit of use? We found an issue with the back button navigation after discarding a draft, which is probably what is happening here.
> 
> * Open composer
> * Type a bit of a message
> * Cancel and hit Discard
> * Back button stops working
> 
> Mind seeing if that same thing happens here? Appreciate it!

Yep, that seems to be what happens… on a fresh launch, the back gesture works again, but if I type a bit of a post in the composer, cancel, and discard, the back gesture doesn't work anymore. Thanks for looking into this!

**Comment by haileyok** [comment]
@TinaRussell Can you try updating again to 1.75 and see if it is working for you now?

**Comment by TinaRussell** [comment]
@haileyok I don't remember updating the app—weird, since I usually have to do it manually—but the "app info" for Bluesky says 1.75.0 now for me, and once I restarted the app, the back gesture problem went away. I tried composing and then discarding a post, and the back gesture still seems to work. So, I guess this is closed, unless the problem recurs for some reason. Thanks!

**Comment by TinaRussell** [comment]
Aw, rats. I've been using the Bluesky app for a while and, indeed, the back gesture stopped working again. I wish I knew what caused the problem this time. I'll let y'alls know if I figure it out.

**Comment by gensakudan** [comment]
I'm not sure if this is the same issue, but I've experienced similar behavior with my phone's back button. It works just fine on a fresh startup, but once I switch accounts within the app, it no longer responds. I have to completely stop the app and then restart it for it to work again.

App version: 1.76.0 (102, production)
Android version: 13

**Comment by mozzius** [comment]
Is this still happening?

**Comment by mozzius** [comment]
Closing as stale

**Comment by TinaRussell** [comment]
Oh, err, yeah, this issue hasn’t happened in a while, and I kept forgetting to close it 😅 Glad to see it’s closed.
